src/AltinnCore/Common/Helpers/AuthenticationHelper.cs

```diff
@@ -84,5 +97,23 @@ public static string GetGiteaSession(HttpContext context, string sessionCookieId
 
             return null;
         }
+
+        /// <summary>
+        /// Creates a HttpClient with nescerry cookies to call the designer APIS from runtime.
+        /// Relevant 
```

Comment:
Feel like this is an incomplete sentence, please remove the start of the sentence or complete it

Reply:
Updated